libvast/src/detail/posix.cpp

```diff
@@ -31,6 +31,7 @@
 #include <string.h>
 #include <unistd.h>
 
+#include <asm-generic/errno-base.h>
```

Comment:
What is this for?

Reply:
Uhm, no idea what that is and how that got there :confused: